content/geometry/general.adoc

```diff
@@ -120,60 +120,63 @@ The following naming conventions apply to {THIS_STANDARD} geometry files:
 === Node structure
 
 Every node structure for a 3D object uses predefined keywords to allow a consistent naming convention and parsing.
-Some keywords are already defined by the standard and more could follow in future updates.
-The user is free to add more keywords for himself, if they are needed.
+Some keywords are already defined by the {THIS_STANDARD} standard and more could follow in future updates.
+Users are free to add more keywords for themself, if they are needed. The following rules apply:
 
-* All components shall be named according to the capital Snake_Case definition, starting with uppercase letters.
+* All components shall be named according to capital snake case definition, starting with uppercase letters.
 * Group nodes (also known as empty nodes or parent nodes) shall have "Grp_" as a prefix.
-* Iterators are added as suffixes.
-  In the documentation, iterator names are written in pointy braces. Example: <type_idx>.
-  In the node names itself, the iterator names are replaced by integer values starting from 0.
+* Iterators shall be added as suffixes.
+  In the documentation, iterator names are written in angled brackets. Example: <type_idx>.
+  In the node names itself, the iterator names are replaced by integer values, starting from 0.
 * Sequence of suffixes: 1: Iterator 2: Type Enumerator 3: Subtype Enumerator.
-* The predefined keywords shall be used for the corresponding asset parts and can be found in the according subchapters.
+* The predefined keywords shall be used for the corresponding asset parts and can be found in the corresponding subchapters.
 
 === Metadata
 
-* Naming of fields shall be according to lowerCamelCase definition, starting with lower letters.
+The following rules for metadata apply:
+
+* Fields shall be named according to the Camel case definition, starting with lowercase letters.
```

Comment:
I would prefer calling it lowerCamelCase as the name directly suggest how to write it, similar to Snake_Case above.

Reply:
Changed accordingly.